### PR TITLE
Move color scale to utility class to avoid code duplication

### PR DESCRIPTION
**Pull request by geofjamg**

### CODE REVIEW

File: commons/src/main/java/eu/itesla_project/commons/util/Colors.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ */
+public class Colors {
+
+    private Colors() {
+    }
+
+    private static final double GOLDEN_RATIO_CONJUGATE = 0.618033988749895;
+
+    public static String[] generateColorScale(int n) {
+        String[] colors = new String[n];
```
Comment (mathbagu) [inline commons/src/main/java/eu/itesla_project/commons/util/Colors.java]: Asser n is positive

File: commons/src/main/java/eu/itesla_project/commons/util/Colors.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            h += GOLDEN_RATIO_CONJUGATE;
+            h %= 1;
+            long[] rgb = hsvToRgb(h, 0.5, 0.95);
+            String hex = String.format("#%02x%02x%02x", rgb[0], rgb[1], rgb[2]).toUpperCase();
+            colors[i] = hex;
+        }
+        return colors;
+    }
+
+    private static long[] hsvToRgb(double h, double s, double v) {
```
Comment (mathbagu) [inline commons/src/main/java/eu/itesla_project/commons/util/Colors.java:38]: Assert h, s and v are positive

File: commons/src/main/java/eu/itesla_project/commons/util/Colors.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.commons.util;
+
+import java.util.Random;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class Colors {
```
Comment (mathbagu) [inline commons/src/main/java/eu/itesla_project/commons/util/Colors.java]: Should be declared final?

File: commons/src/main/java/eu/itesla_project/commons/util/Colors.java (hunk 1 above)
Comment (geofjamg) [reply commons/src/main/java/eu/itesla_project/commons/util/Colors.java]: done

File: commons/src/main/java/eu/itesla_project/commons/util/Colors.java (hunk 2 above)
Comment (geofjamg) [reply commons/src/main/java/eu/itesla_project/commons/util/Colors.java:38]: Useless, it is private method

File: commons/src/main/java/eu/itesla_project/commons/util/Colors.java (hunk 3 above)
Comment (geofjamg) [reply commons/src/main/java/eu/itesla_project/commons/util/Colors.java]: done

**Pull request merged**